### PR TITLE
Kiosk documentation reorganization

### LINKED ISSUE #88: Add a section to FAQs for help and support
Direct to kiosk documentation and an appropriate issues page on Github

### LINKED ISSUE #90: Add instructions about supported input data types

### LINKED ISSUE #91: Rewrite readme page

### PR DESCRIPTION
**Pull request by msschwartz21**
- Setup read the docs for the frontend
- Add frontend as a subproject of the kiosk on RTD
- Setup universal sidebar for all kiosk-related projects on RTD

---

Fixes #88 
Fixes #90 
Fixes #91

### CODE REVIEW

**Comment by willgraf** [comment]
Is there a way that this functionality does not introduce Python into our JS project?

**Comment by msschwartz21** [comment]
I don't think that's possible. At minimum we need the conf.py file to run sphinx. Unfortunately I don't know of any good ways to pull in content without setting up RTD and sphinx.

**Comment by msschwartz21** [comment]
I'm going to close this pull request because there are a couple additional changes that need to be implemented

**Comment by msschwartz21** [comment]
This PR is ready to be merged along with the other documentation PRs.

File: README.md [hunk 1]
```diff
@@ -8,17 +8,10 @@ DeepCell graphical user interface built using React, Babel, Webpack. Run with `n
 
 The job types are defined as an environment variable `JOB_TYPES`, which evaluates to `"segmentation,tracking"` by default. This can easily be extended by just adding to this string, for example, `"segmentation,tracking,spot detection"`. These values are parsed into a list and populated into the drop-down with the route `/jobtypes`. Each job type value is also the exact value of the Redis queue used by the corresponding consumer (e.g. `"segmentation"` is both the job type, and the queue used for the `segmentation-consumer`).
 
-## Front End React Hierarchy Diagram
-![Alt text](./docs/kiosk-frontend-react.png)
+## Front End React Hierarchy
 
 The `App` Component in `/src` folder is the first parent Component that contains all other components. The `App` Component is directly referenced by the `index.js` file inside the `./src` folder. The `index.js` file is the entry point for Webpack (see `webpack.config.js` to bundle the app's data and serve it via `public/index.html` using a plugin called `HtmlWebpackPlugin`. The app also uses Babel via `.babelrc`, which is accessed within Webpack (press `Command + Shift + .` to be able to view invisible files in Mac Finder).
```
Comment (willgraf) [inline README.md]: Looks like you didn't write this, but can you remove "(press `Command + Shift + .` to be able to view invisible files in Mac Finder)."? That seems pretty un-related to the actual project.

File: README.md [hunk 2]
```diff
@@ -8,17 +8,10 @@ DeepCell graphical user interface built using React, Babel, Webpack. Run with `n
 
 The job types are defined as an environment variable `JOB_TYPES`, which evaluates to `"segmentation,tracking"` by default. This can easily be extended by just adding to this string, for example, `"segmentation,tracking,spot detection"`. These values are parsed into a list and populated into the drop-down with the route `/jobtypes`. Each job type value is also the exact value of the Redis queue used by the corresponding consumer (e.g. `"segmentation"` is both the job type, and the queue used for the `segmentation-consumer`).
```
Comment (willgraf) [inline README.md]: Again, not your work but can you remove the commm in "both the job type, and the queue".

**Comment by willgraf** [comment]
I noticed that you fix #90 in this PR. Maybe we can also fix #88 as well, with a quick edit to the FAW.

**Pull request merged**